source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
+a different programming language, select from the ones on our
+:driver:`drivers documentation page <>`.
 
-First, check the version of Node:
+MongoDB Atlas is a fully-managed cloud database service that hosts your data
+on MongoDB instances. We show you how to get started with your own free (no
+credit card required) instance in this guide.
 
-.. code-block:: sh
+Follow the steps below to connect your Node.js application with a MongoDB
+instance.
 
-   node -v
+Set up Your Project
+-------------------
 
-You should see output similar to the following:
+Install Node and NPM
+~~~~~~~~~~~~~~~~~~~~
 
-.. code-block:: sh
-
-   v12.14.1
-
-Next, check the version of NPM:
-
-.. code-block:: sh
-
-   npm -v
-
-You should see output similar to the following:
-
-.. code-block:: sh
-
-   6.13.4
-
-If you do not see any output or encounter an error, you will need to
-install Node and NPM. Visit `nodejs.org <https://nodejs.org>`_ for
-installation instructions.
+Ensure your system has Node.js version 8 or later and a compatible version of
+NPM (Node Package Manager) installed. For more information on how to check
+your version of Node and NPM and installation instructions for your
+system, see `downloading and installing Node.js and npm
+<https://docs.npmjs.com/downloading-and-installing-node-js-and-npm>`_.
 
 Initialize the Project
-----------------------
+~~~~~~~~~~~~~~~~~~~~~~
 
-Now that our system meets the Node driver's requirements, we can
-create a new project.
+After you verify that you installed the required version of Node and a
+compatible version of NPM, create a new project.
 
-First, create a directory for your project:
+First, create a directory for your project in your command line interface:
 
-.. code-block:: sh
+.. code-block:: none
 
-  mkdir node_quickstart
+   mkdir node_quickstart
 
 Then, navigate into that directory so you can work directly with your
 project's files:
 
-.. code-block:: sh
+.. code-block:: none
 
   cd node_quickstart
 
-Finally, initialize npm:
+Next, set up NPM for your project by running the following command.
+This command creates a file called ``package.json``:
 
-.. code-block:: sh
+.. code-block:: none
 
   npm init -y
 
 .. admonition:: Why the -y?
   :class: note
 
-  Specifying ``-y`` tells NPM to use default values. You can omit the
-  ``-y`` flag to proceed through an interactive selection process for
-  common project settings.
+  If you specify the ``-y`` option in the command, NPM automatically
+  accepts the default values for the command. Omit the ``-y`` flag to
+  interactively select your project settings.
 
 Add MongoDB as a Dependency
----------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Next, add the Node MongoDB driver as a project dependency. Use the
-``--save`` flag to save the dependency to the ``package.json`` file that
-was automatically generated in the previous step.
+Next, add the offical Node.js MongoDB driver to your project dependencies. Use
```

Comment:
Typo: "offical" should say "official"

the full sentence should read:
"Next, add the official Node.js MongoDB driver to your project dependencies. Use ..."